packages/default-theme/components/checkout/steps/OrderReviewStep.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div>
+    <SfHeading
+      title="4. Order review"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <SfAccordion first-open class="accordion mobile-only">
+      <SfAccordionItem header="Personal Details">
+        <PersonalDetailsSummary @click:edit="$emit('click:edit', 0)" />
```

Comment:
you can use constants as you did in PersonalDetailsSummary:
```@click="$emit('click:edit', CHECKOUT_STEPS.PERSONAL_DETAILS)```

Reply:
yeah, I planned to create an issue for that :)

Reply:
https://github.com/DivanteLtd/shopware-pwa/issues/610